src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -232,6 +239,15 @@ int ExecuteManagedAssembly(
     std::string tpaList;
     AddFilesFromDirectoryToTpaList(clrFilesAbsolutePath, tpaList);
 
+#if defined(__LINUX__)
+    void* coreclrTPLib = dlopen(coreClrTPproviderDllPath.c_str(),RTLD_NOW | RTLD_GLOBAL);
+    if (coreclrTPLib == nullptr)
+    {
+        char *error_msg = dlerror();
+        fprintf(stderr, "Event Tracing is disabled.\n dlopen failed to load tracepoint provider:%s, with error:%s\n",coreClrTpProviderDll,error_msg);
```

Comment:
I was banking on the thing being annoying to ensure maximum test coverage. But if you guys feel strongly against it, will remove it.